scripts/gluster-block/setup-gluster-block-glusto.yml

```diff
@@ -234,19 +235,14 @@
         repo: https://github.com/open-iscsi/tcmu-runner
         dest: /root/tcmu-runner
 
-  - replace:
-      path: /root/tcmu-runner/target_core_user_local.h
-      regexp: '^#include <linux/uio\.h>'
-      replace: '#include <sys/uio.h>'
-
   - name: Install tcmu-runner
     command: 'chdir=/root/tcmu-runner {{ item }}'
     with_items:
     -  cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr -Dwith-rbd=false -Dwith-qcow=false -DSUPPORT_SYSTEMD=ON -DCMAKE_BUILD_TYPE=Debug
     -  make install
 
   - name: Create /etc/target if it does not exist
-    command: 'if [ ! -d "/etc/target" ]; then mkdir /etc/target ; fi'
```

Comment:
Uh, do you just want a file there? Pretty sure the ansible file module can do this for you.

Reply:
I am checking for /etc/target directory. If it doesn't exist then create it.

Reply:
Yep, there's an ansible module to do exactly that.

Reply:
@nigelbabu  updated the PR